releases/release-1.24/release-notes/release-notes-draft.json

```diff
@@ -2283,19 +2630,51 @@
     "duplicate": true,
     "duplicate_kind": true
   },
-  "107552": {
```

Comment:
What does removal of a release note mean?

Reply:
As the `"107552": {` is present as well line 2665 I think it is just how git handles it. Sometimes it deletes it from one line, add a block below and re-add the deleted line later.